ks-prometheus/components/whizard-telemetry-mixin/rules/gpu.libsonnet

```diff
@@ -0,0 +1,499 @@
+{
+  _config+:: {
+    cambriconMLUMonitoringSelector: 'job="mlu-monitoring"',
+    nvidiaGPUMonitoringSelector: 'job="nvidia-dcgm-exporter"',
+    ascendNPUMonitoringSelector: 'job="npu-exporter"',
+    kubeStateMetricsSelector: 'job="kube-state-metrics"',
+    clusterLabel: 'cluster',
+  },
+
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: "whizard-telemetry-cambricon-mlu.rules",
+        rules: [
+          {
+            record: 'node_namespace_pod_container:container_gpu_utilization',
+            expr: |||
+                sum by (%(clusterLabel)s, node, namespace, pod, container) (
+                    mlu_utilization{%(cambriconMLUMonitoringSelector)s} / 100
+                  * on (uuid) group_left (namespace, pod, container, node)
```

Comment:
may be on `uuid` and `cluster`

Reply:
doesn't matter because uuids don't usually collide

Reply:
Fixed.